test/lib/rule-finder.js

```diff
@@ -77,57 +85,107 @@ describe('rule-finder', function() {
 
   it('specifiedFile (relative path) is passed to the constructor', function() {
     var ruleFinder = getRuleFinder(specifiedFileRelative)
-    assert.deepEqual(ruleFinder.getUnusedRules(), ['baz-rule', 'react/bar-rule', 'react/baz-rule', 'react/foo-rule'])
+    assert.deepEqual(ruleFinder.getUnusedRules(), [
+      'baz-rule',
+      'react/bar-rule',
+      'react/baz-rule',
+      'react/foo-rule',
+      'something/two-rule',
```

Comment:
@ta2edchimp I tested it in another project as I was working on this, and yes, for configuration rules take only the plugin name and not the scope name. Wasn't my first guess.